diff --git a/samples/Samples.h b/samples/Samples.h
index 6b721a8d6..0b3726090 100644
--- a/samples/Samples.h
+++ b/samples/Samples.h
@@ -50,7 +50,7 @@ extern "C" {
 #define VIEWER_DATA_CHANNEL_MESSAGE "This message is from the KVS Viewer"
 
 /* Uncomment the following line in order to enable IoT credentials checks in the provided samples */
-//#define IOT_CORE_ENABLE_CREDENTIALS  1
+#define IOT_CORE_ENABLE_CREDENTIALS  1
 
 typedef enum {
     SAMPLE_STREAMING_VIDEO_ONLY,
diff --git a/samples/kvsWebRTCClientMasterGstreamerSample.c b/samples/kvsWebRTCClientMasterGstreamerSample.c
index 185859bd5..aa6fa13b3 100644
--- a/samples/kvsWebRTCClientMasterGstreamerSample.c
+++ b/samples/kvsWebRTCClientMasterGstreamerSample.c
@@ -151,10 +151,17 @@ PVOID sendGstreamerAudioVideo(PVOID args)
                     "video/x-h264,stream-format=byte-stream,alignment=au,profile=baseline ! appsink sync=TRUE emit-signals=TRUE name=appsink-video",
                     &error);
             } else {
+                //pipeline = gst_parse_launch(
+                //    "autovideosrc ! queue ! videoconvert ! video/x-raw,width=1280,height=720,framerate=[30/1,10000000/333333] ! "
+                //    "x264enc bframes=0 speed-preset=veryfast bitrate=512 byte-stream=TRUE tune=zerolatency ! "
+                //    "video/x-h264,stream-format=byte-stream,alignment=au,profile=baseline ! appsink sync=TRUE emit-signals=TRUE name=appsink-video",
+                //    &error);
                 pipeline = gst_parse_launch(
-                    "autovideosrc ! queue ! videoconvert ! video/x-raw,width=1280,height=720,framerate=[30/1,10000000/333333] ! "
+                    "shmsrc socket-path=/tmp/visible is-live=true do-timestamp=true ! "
+                    "video/x-raw, format=(string)BGR, width=(int)1920, height=(int)1080, framerate=(fraction)5/1 ! videoconvert ! "
                     "x264enc bframes=0 speed-preset=veryfast bitrate=512 byte-stream=TRUE tune=zerolatency ! "
-                    "video/x-h264,stream-format=byte-stream,alignment=au,profile=baseline ! appsink sync=TRUE emit-signals=TRUE name=appsink-video",
+                    "video/x-h264,stream-format=byte-stream,alignment=au,profile=baseline ! "
+                    "appsink sync=TRUE emit-signals=TRUE name=appsink-video",
                     &error);
             }
             break;
